src/Components/Blueprints/BlueprintsSideBar.tsx

```diff
@@ -141,30 +124,37 @@ const BlueprintsSidebar = ({
         {blueprintsTotal > 0 &&
           blueprints?.map((blueprint: BlueprintItem) => (
             <StackItem key={blueprint.id}>
-              <BlueprintCard
-                blueprint={blueprint}
-                selectedBlueprint={selectedBlueprint}
-                setSelectedBlueprint={setSelectedBlueprint}
-              />
+              <BlueprintCard blueprint={blueprint} />
             </StackItem>
           ))}
       </Stack>
     </>
   );
 };
 
-const BlueprintSearch = ({
-  filter,
-  setFilter,
-  blueprintsTotal,
-}: blueprintSearchProps) => {
+const BlueprintSearch = ({ blueprintsTotal }: blueprintSearchProps) => {
+  const blueprintsSearchQuery = useAppSelector((state) =>
```

Comment:
Would you mind changing this to `blueprintSearchInput`? I find it easier to understand the code when the name of the variable in the React component is the same as it is in the store.